src/experiment_prototype/experiment_slice.py

```diff
@@ -417,21 +417,32 @@ def check_intt(cls, intt, values):
 
     @validator('tx_antennas')
     def check_tx_antennas(cls, tx_antennas):
-        if not tx_antennas:
-            return [i for i in options.main_antennas]
+        if tx_antennas is None:
+            tx_antennas = [i for i in options.tx_main_antennas]
+        for ant in tx_antennas:
+            if ant not in options.tx_main_antennas:
+                raise ValueError(f"TX antenna {ant} not specified in config file")
         return tx_antennas
 
     @validator('rx_main_antennas')
     def check_rx_main_antennas(cls, rx_main_antennas):
-        if not rx_main_antennas:
-            return [i for i in options.main_antennas]
+        if rx_main_antennas is None:
+            rx_main_antennas = [i for i in options.rx_main_antennas]
+        if len(rx_main_antennas) == 0:
+            raise ValueError("Must have at least one main antenna for RX")
+        for ant in rx_main_antennas:
+            if ant not in options.rx_main_antennas:
+                raise ValueError(f"RX main antenna {ant} not specified in config file")
```

Comment:
There are not, that's a good idea however!

Reply:
The second one is tough to test, since all our config files use all possible antennas given the `main_antenna_count` field. The solution would be to test different possible config files, like with Theo's comment down below.

Reply:
Upon further thought, I don't even know how we could easily test that last condition with an automated test. You would have to make a config file that doesn't use all the possible antennas, then make an experiment that specifies an antenna in `rx_main_antennas` that isn't specified in the config file. The simple solution that I can think of is to test with the lab setup, but if things change in the future that doesn't ensure that the testing will catch new bugs. This may be the impetus for an integration test suite between the config file and experiment tests.

Reply:
Nevermind, I found a way to do this within the new config file test suite.